conjureup/controllers/lxdsetup/gui.py

```diff
@@ -1,16 +1,9 @@
-from conjureup.app_config import app
-from conjureup.ui.views.lxdsetup import LXDSetupView
-
 from . import common
 
 
 class LXDSetupController(common.BaseLXDSetupController):
     def render(self):
-        if len(self.ifaces) == 1:
-            return self.setup(self.ifaces[0])
-
-        view = LXDSetupView(app.provider, self.setup)
```

Comment:
Why aren't we showing the setup view now?  Does it need more work before it's useful?